src/NServiceBus.NHibernate/SharedSession/NonSharedConnectionStorageSessionProvider.cs

```diff
@@ -12,16 +12,6 @@ public NonSharedConnectionStorageSessionProvider(SessionFactoryProvider sessionF
             this.sessionFactoryProvider = sessionFactoryProvider;
         }
 
-        public IStatelessSession OpenStatelessSession()
```

Comment:
Since I disabled sharing of connection in this session, I could remove that and uses the `ISessionFactory` directly.